dodola/tests/test_services.py

```diff
@@ -32,6 +40,37 @@ def _datafactory(x, start_time="1950-01-01"):
     return out
 
 
+def _gcmfactory(x, start_time="1950-01-01"):
+    """Populate xr.Dataset with synthetic data for testing"""
```

Comment:
Please change, or elaborate docstr. On glance, it's not clear how this is different from `_datafactory()`, or why it's needed.

Reply:
good point - I've updated the docstring.